### DISCUSSION TITLE
Either Multiple Input or single input option

### DISCUSSION THREAD

**Discussion by swayamtejwani** [dmuy/duDatepicker#55] [Q&A]
Hi @dmuy 
Thanks for such a great plugin. I have a situation here where user is having option to either choose one way or round trip option. If they select round trip, the second date should start from the first input selected. How can we achieve that?
Or is that possible we can have 2 datepickers, one for only one way and when user selcts round trip then we show date range datepicker, am kind of stuck here. Have tried different ways using jquery but unable to achieve it.

Below is the code for referece, let me know.
`
      duDatepicker("#quickcab_form_departure_date", {
        format: 'mm/dd/yyyy', range: false, clearBtn: true,
        minDate:'today',
        events: {
          dateChanged: function (data) {
           console.log("date: ",data.date);
           //localStorage.setItem("mindate", data.date);
           //jQuery('#quickcab_form_return_departure_date').attr('data-mindate',data.date);
           // duDatepicker("#quickcab_form_return_departure_date", 'setMinDate', data.date);
          },
         
        }
      })

      duDatepicker("#quickcab_form_return_departure_date", {
              format: 'mm/dd/yyyy', range: false, clearBtn: true,
              // minDate:localStorage.getItem("mindate"),
              events: {
                dateChanged: function (data) {
                  console.log("date: ",data.date);
                },
              }
            })
`

**Comment by dmuy** [comment]
Hi @swayamtejwani,

I'm so sorry for the late response, if you haven't found a solution yet. You can try [this](https://github.com/dmuy/duDatepicker/wiki/Usage#using-two-input-elements) example from the wiki - although this uses a total of 3 inputs.
